pwa/src/components/componentCard/ComponentCard.module.css

```diff
@@ -0,0 +1,92 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  border: var(--web-app-card-border-color);
+  padding-inline-start: var(--web-app-size-md);
+  padding-inline-end: var(--web-app-size-md);
+  padding-block-start: var(--web-app-size-md);
+  padding-block-end: var(--web-app-size-md);
+}
+
+.container > *:not(.titleLink):not(:last-child) {
+  margin-block-end: var(--web-app-size-md);
+}
+
+.titleLink {
+  font-size: var(--web-app-font-size-lg);
+  --denhaag-link-icon-gap: var(--web-app-size-xs);
+}
+
+.titleLink > :first-child > span > svg {
+  width: var(--web-app-font-size-lg);
+  height: var(--web-app-font-size-lg);
+}
+
+.titleLink > :first-child {
+  display: flex;
+}
+
+.description {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.layerTags {
+  display: flex;
+}
+
+.layerTags > *:not(:last-child) {
+  margin-inline-end: var(--web-app-size-xs);
+}
+
+.tags > div {
+  margin-block-end: var(--web-app-size-xs);
+}
+
+.tags > *:not(:last-child) {
+  margin-inline-end: var(--web-app-size-xs);
+}
+
+/* Colors of the 5 layers */
+.interactie {
+  --web-app-tag-background-color: #1a75ff;
+  --web-app-tag-color: var(--web-app-color-white);
+}
+.interactieCategorie {
+  --web-app-tag-background-color: #cfe2ff;
+}
+
+.proces {
+  --web-app-tag-background-color: #dd3c49;
+  --web-app-tag-color: var(--web-app-color-white);
+}
+.procesCategorie {
+  --web-app-tag-background-color: #f8d7da;
+}
+
+.integratie {
+  --web-app-tag-background-color: #efc025;
+  --web-app-tag-color: var(--web-app-color-white);
+}
+.integratieCategorie {
+  --web-app-tag-background-color: #fff3cd;
+}
+
+.services {
+  --web-app-tag-background-color: #69b090;
+  --web-app-tag-color: var(--web-app-color-white);
+}
+.servicesCategorie {
+  --web-app-tag-background-color: #d1e7dd;
+}
+
+.data {
+  --web-app-tag-background-color: #7a51c8;
+  --web-app-tag-color: var(--web-app-color-white);
+}
+.dataCategorie {
+  --web-app-tag-background-color: #e2d9f3;
+}
```

Comment:
Every layer should have a design token.

E.g.:

```
--web-app-layer-color-integration: #000000;
--web-app-layer-color-intraction: #000000;
...
```

The corresponding categories could be like this:

```
color: var(--web-app-layer-color-integration)50;
```